### ISSUE TITLE
v4.0.0-beta.12: Unknown or invalid refresh token thrown from middleware

### ISSUE THREAD

**Issue by chris-erickson** [auth0/nextjs-auth0#1851]
### Checklist

- [X] The issue can be reproduced in the [nextjs-auth0 sample app](https://github.com/auth0-samples/auth0-nextjs-samples/tree/main/Sample-01) (or N/A).
- [X] I have looked into the [Readme](https://github.com/auth0/nextjs-auth0#readme), [Examples](https://github.com/auth0/nextjs-auth0/blob/main/EXAMPLES.md), and [FAQ](https://github.com/auth0/nextjs-auth0/blob/main/FAQ.md) and have not found a suitable solution or answer.
- [X] I have looked into the [API documentation](https://auth0.github.io/nextjs-auth0/) and have not found a suitable solution or answer.
- [X] I have searched the [issues](https://github.com/auth0/nextjs-auth0/issues) and have not found a suitable solution or answer.
- [X] I have searched the [Auth0 Community](https://community.auth0.com) forums and have not found a suitable solution or answer.
- [X] I agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).

### Description

I followed the middleware setup instructions. If I leave a web app running overnight with inactivity, routes start triggering this error in the middleware.

### Reproduction

```
ResponseBodyError: server responded with an error in the response body
    at async middleware (middleware.ts:8:18)
   6 |
   7 | export async function middleware(request: NextRequest) {
>  8 |   const authRes = await auth0.middleware(request);
     |                  ^
   9 |
  10 |   // authentication routes — let the middleware handle it
  11 |   if (request.nextUrl.pathname.startsWith("/auth")) { {
  code: 'OAUTH_RESPONSE_BODY_ERROR',
  error: 'invalid_grant',
  status: 403,
  error_description: 'Unknown or invalid refresh token.'
```

### nextjs-auth0 version

v4.0.0-beta.12

### Next.js version

15.1.1

### Node.js version

v20.17.0

**Comment by guabu** [comment]
Hey @chris-erickson 👋 Thanks for the feedback! We're addressing this as part of another set of fixes around fetching and refreshing access tokens (linked above).

We'll be moving away from automatically refreshing access tokens in the middleware so this should fix the issue you're reporting.

**Comment by guabu** [comment]
This should be fixed in the latest release (`4.0.0-beta.13`). We've moved the access token refresh handling from the middleware to the `getAccessToken` handler so you should no longer be running into this issue after upgrading. Thanks for reporting this!
